apps/prairielearn/src/lib/question-submission.ts

```diff
@@ -73,12 +73,14 @@ export async function processSubmission(
 
   if (req.body.__action === 'grade') {
     const overrideRateLimits = !options.studentSubmission;
+    const overrideRealTimeGradingDisabled = !options.studentSubmission;
```

Comment:
nit: This may just be me, but I find the `ignore...` easier to parse/understand in my head rather than `override...`

Reply:
IMO that'd be a nice change. Are you comfortable saving that for a PR where we do this new option and `overrideRateLimits` at the same time?

Reply:
Yes, that sounds good.